src/sentry/api/serializers/snuba.py

```diff
@@ -146,7 +146,10 @@ def zerofill(data, start, end, rollup, allow_partial_buckets=False):
         except IndexError:
             pass
 
-        rv.append((key, []))
+        if fill_default is None:
```

Comment:
nit: Can this be move outside of the loop?

Reply:
Doing this at the beginning of the function should be fine right